lib/Controller/ChatController.php

```diff
@@ -258,7 +258,7 @@ public function receiveMessages(int $lookIntoFuture, int $limit = 100, int $last
 
 			$parentId = (int) $comment->getParentId();
 			if ($parentId !== 0) {
-				$parentIds[$id] = $comment->getParentId();
+				$parentIds[$id] = $parentId;
```

Comment:
I think it needs a string later on, because the comments API is based on string IDs